mava/systems/tf/mad4pg/training.py

```diff
@@ -469,15 +482,15 @@ def __init__(
         )
 
     # Forward pass that calculates loss.
-    def _forward(self, inputs: Any) -> None:
+    def _forward(self, inputs: reverb.ReplaySample) -> None:
         """Trainer forward pass
 
         Args:
             inputs (Any): input data from the data table (transitions)
         """
 
         # TODO: Update this forward function to work like MAD4PG
```

Comment:
Do we want to keep this TODO?

Reply:
Not sure, @DriesSmit do we still need this?